docroot/themes/custom/uids_base/scss/views/view-display-id-people.scss

```diff
@@ -2,11 +2,20 @@
 @import "uids/assets/scss/_utilities.scss";
 
 .view-id-people {
+  .views-exposed-form {
+    .form--inline {
+      @include breakpoint(sm) {
+        flex-wrap: unset;
+      }
+    }
+  }
```

Comment:
Is this something that should be applied more broadly than just to this one view?